.travis.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  - docker
+  
+script:
+  - docker build --tag todo-app-dev --target development .
+  - docker build --tag todo-app-test --target test .
+  - docker run -e API_KEY -e SERVER_TOKEN -e BOARD_ID todo-app-test
```

Comment:
I don't expect the tests to need BOARD_ID because the e2e tests should be creating a temporary board.

Reply:
You're running all of your tests with real env vars. This is a bit of extra risk for no reason. Especially when passing in a real board_id, as you could end up with automated tests modifying your "real" data. You aren't copying the .env.test into the image, so the integration tests aren't loading the fake env vars.

If you change the `CMD` to `ENTRYPOINT` in the test stage in your Dockerfile, then you could split this line into two commands:

```
- docker run --env-file .env.test todo-app-test todo_app/tests
- docker run -e API_KEY -e SERVER_TOKEN todo-app-test todo_app/tests_e2e
```